se.sjoerd.Graphs.json

```diff
@@ -0,0 +1,40 @@
+{
+    "app-id" : "se.sjoerd.Graphs",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "graphs",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--device=dri"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
```

Comment:
This entire list seems unused? You can remove it and look in `files` to see whats actually extra, if anything.

Reply:
This list was part of the manifest template in GNOME Builder, but I'll take a look if any of this is necessary.

Reply:
This is a minor detail so you can look at it later.